### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by Rexogamer**
-add project specific requirements 
-remove WIP tag for python projects

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,19 @@ Python script for creating new projects in the desired local directory, with a G
...
 - Python 3.4+
 - Git
-- npm
-- A text editor (i.e. VS Code, Atom, etc.)
+- (Optional but recommended) A text editor (i.e. VS Code, Atom, etc.)
+In addition:
+#### Node/React
+- NPM
+- (React) create-react-app
+- (React) NPX
```
Comment (jarodburchill) [inline README.md]: comes with npm, not required

File: README.md [hunk 2]
```diff
@@ -3,10 +3,19 @@ Python script for creating new projects in the desired local directory, with a G
...
+#### Universal
 - Python 3.4+
 - Git
-- npm
-- A text editor (i.e. VS Code, Atom, etc.)
+- (Optional but recommended) A text editor (i.e. VS Code, Atom, etc.)
+In addition:
+#### Node/React
+- NPM
+- (React) create-react-app
```
Comment (jarodburchill) [inline README.md]: comes with npm, not required

File: README.md [hunk 3]
```diff
@@ -3,10 +3,19 @@ Python script for creating new projects in the desired local directory, with a G
...
 Contact us [on Discord.](https://discord.gg/eqWstJu)
 ## Requirements:
+#### Universal
 - Python 3.4+
 - Git
-- npm
-- A text editor (i.e. VS Code, Atom, etc.)
+- (Optional but recommended) A text editor (i.e. VS Code, Atom, etc.)
+In addition:
+#### Node/React
```
Comment (jarodburchill) [inline README.md]: Node.js/React/Express.js

**Review by jarodburchill — CHANGES_REQUESTED** [review]
Changes required

File: README.md (hunk 2 above)
Comment (jarodburchill) [reply README.md]: this needs to go too..... it is part of npx which comes with npm https://reactjs.org/docs/create-a-new-react-app.html

File: README.md [hunk 4]
```diff
@@ -3,10 +3,17 @@ Python script for creating new projects in the desired local directory, with a G
...
 Contact us [on Discord.](https://discord.gg/eqWstJu)
 ## Requirements:
+#### Universal
 - Python 3.4+
 - Git
-- npm
-- A text editor (i.e. VS Code, Atom, etc.)
+- (Optional but recommended) A text editor (i.e. VS Code, Atom, etc.)
+In addition:
+#### Node/React/Express
```
Comment (jarodburchill) [inline README.md]: .js on end of Node and Express

File: README.md (hunk 2 above)
Comment (Rexogamer) [reply README.md]: Do you mean that the script comes with NPM, or that CRA comes with NPM

Comment (jarodburchill) [reply README.md]: Where did NPM go?

**Review by jarodburchill — CHANGES_REQUESTED** [review]

**Comment by Rexogamer** [comment]
All good now?

File: README.md [hunk 5]
```diff
@@ -3,10 +3,17 @@ Python script for creating new projects in the desired local directory, with a G
...
 Contact us [on Discord.](https://discord.gg/eqWstJu)
 ## Requirements:
+#### Universal
 - Python 3.4+
 - Git
-- npm
-- A text editor (i.e. VS Code, Atom, etc.)
+- (Optional but recommended) A text editor (i.e. VS Code, Atom, etc.)
+In addition:
+#### NodeJS/React/ExpressJS
```
Comment (jarodburchill) [inline README.md]: **.js**

**Review by jarodburchill — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (jarodburchill) [reply README.md]: @Rexogamer exactly like I typed it here

**Pull request merged**